openeo/subscription.js

```diff
@@ -11,6 +11,13 @@ var Subscription = {
 		this.websocketserver = new WebSocket.Server({noServer: true});
 		console.log('INFO: Subscriptions loaded - WebSocket Server started.');
 
+		setInterval(() => {
```

Comment:
That probably shouldn't be in here?

Reply:
Yeah, that's just a debugging remnant. I left it in there because I thought it'd still be handy for further development, but I'm totally fine if we scrap it out of the main repo.